### PR TITLE
RESTWS-801:  Modifiers should be declared in the correct order

### PR DESCRIPTION
**Pull request by kdaud**
## Issue I worked on
see https://issues.openmrs.org/browse/RESTWS-801

## Checklist: I completed these to help reviewers :)

- [ x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by sacull** [comment]
@kdaud Look again at this PR and think if everything is fine with it.

**Comment by kdaud** [comment]
Am not sure why exactly some checks are failing but am working out still to see that its fixed

**Comment by sacull** [comment]
1. If the tests fail and the code does not work, why are you reporting the PR as ready to be reviewed on JIRA?
2. PR title and ticket content: "Modifiers should be declared in the correct order", ale what are you really doing and commit description: "Removed unused imports from files". Do you still think the point is that the test fails?

**Comment by HerbertYiga** [comment]
commit message has a different ticket Id than that of the ticket description ie RESTWS-795 and RESTWS-801

**Comment by kdaud** [comment]
> commit message has a different ticket Id than that of the ticket description ie RESTWS-795 and RESTWS-801

Am working it out  and soon am updating the PR

**Comment by sacull** [comment]
You can additionally remove the indicated spaces in the PR description. When you do that, magic begins. ;)
image

**Comment by kdaud** [comment]
> 1. If the tests fail and the code does not work, why are you reporting the PR as ready to be reviewed on JIRA?
> 2. PR title and ticket content: "Modifiers should be declared in the correct order", ale what are you really doing and commit description: "Removed unused imports from files". Do you still think the point is that the test fails?



> You can additionally remove the indicated spaces in the PR description. When you do that, magic begins. ;)
> image

Have figured out the issue and have resolved it

**Comment by sacull** [comment]
Okay, I'll try to present the problem to you differently. Please write in one sentence what changes does this PR introduce?

**Comment by kdaud** [comment]
> > Have figured out the issue and have resolved it
> 
> Okay, I'll try to present the problem to you differently. Please write in one sentence what changes does this PR introduce?

Refactoring modifiers that were not properly coded in as per Java standard order

**Comment by sacull** [comment]
> Refactoring Java modifiers that were not properly coded as per the standard order convention

So why do you delete imports in this PR?
image

**Comment by kdaud** [comment]
> > Refactoring Java modifiers that were not properly coded as per the standard order convention
> 
> So why do you delete imports in this PR?
> image

Have found out i didn't switch to the correct branch while working on this ticket. Let me work on this concern. Thanks @sacull for being patient with me, am a newbie with this tech But passionate with learning open source software

File: omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/LocationAttributeSearchHandler1_9.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
 @Component
 public class LocationAttributeSearchHandler1_9 implements SubResourceSearchHandler {
 	
-	private final static SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
+	private static  final   SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
```
Comment (sacull) [inline omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/LocationAttributeSearchHandler1_9.java:34]:
```suggestion
	private static final SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
```
You have too many spaces here.

File: omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
  * {@link RestConstants} for 1.9 resources.
  */
 public class RestConstants1_9 {
-
-	private RestConstants1_9(){}
+	
+	private RestConstants1_9() {
+	}
```
Comment (sacull) [inline omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java:20]: Is the formatting changes part of this ticket? It's very good that you want the code to look nice, but changes to files that are out of ticket range shouldn't be done.

File: omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/LocationAttributeSearchHandler1_9.java (hunk 1 above)
Comment (kdaud) [reply omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/LocationAttributeSearchHandler1_9.java:34]: Am working out to resolve the concern

File: omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java (hunk 2 above)
Comment (kdaud) [reply omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java:20]: I didn't make changes in those modified files But here is my findings in respect to those changes:
mvn clean install  allows the automatic code formatter to format code according to the OpenMRS conventions. That’s probably why those files are modified by that process, and the committer should likely commit those changes to the main repo eventually.

Comment (sacull) [reply omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java:20]: Yes, maybe you are right.

**Review by sacull — APPROVED** [review]
It looks good to me. ;)

**Pull request merged**